### DISCUSSION TITLE
How to send modbus write packets on fuxa

### DISCUSSION THREAD

**Discussion by GiveMeRansom** [frangoteam/FUXA#1172] [Q&A]
Hello all, I currently have a modbus server that is connected to the fuxa. How to send modbus write requests to the modbus server. Thanks in advance for answering

**Comment by unocelli** [comment]
Hi
https://youtu.be/THVGn-I2kng?si=B7iBy7S0CG1XagMR
https://youtu.be/OI46d0Ckntw?si=rrSmtJNYX3_6k8KX
https://youtu.be/aHFseA1HVsk?si=2tQ6_S0PfJZVOSpK

**Reply by NarutSH** [reply]
Hi i already watch your 3 videos, but i have not found the solution that i can send write request to modbus server.

can you please provide me the solution ?
